app/js/newTest.js

```diff
@@ -1,10 +1,26 @@
-document.querySelector('.new-test-button').addEventListener('click', function (x) {
-        let inputLength = document.querySelector('#addTestName').value.length
-        if (inputLength < 1 || inputLength > 255) {
-            alert('Computer says no.')
-        } else {
-            alert('Computer says yes.')
-        }
+var submit = document.querySelector('.new-test-button')
```

Comment:
Can we make this more symantic? replace with 'submitNewTest'

Reply:
Hmmm this falls in line with the pre-existing code, if we change this then we'd have to change the one for the new question area. 

Not sure which bits of JS depend on that, so I think it would make sense to leave it as is.